src/gsy_e/gsy_e_core/export.py

```diff
@@ -444,20 +511,28 @@ def plot_all_unmatched_loads(self):
         output_file = os.path.join(plot_dir, f"unmatched_loads_{root_name}.html")
         PlotlyGraph.plot_bar_graph(plot_desc, output_file)
 
-    def plot_ess_soc_history(self, area, subdir):
+
+class PlotESSSocHistory:
```

Comment:
```suggestion
class PlotEssSocHistory:
```
OR
```suggestion
class PlotESSSOCHistory:
```

Reply:
Since you like it I will change it, however 7 capital letters in a row look weird to me.

Reply:
`PlotEssSocHistory `  was the alternate proposal

Reply:
Indeed, misread it, does not look weird enough to me though to go through another iteration. Leaving it as is